### PR TITLE
Fixing the FCM categories

### LINKED ISSUE #1337: fcm categories need to be updated
The categories available in the CMS aren't the categories outlined in the spec. They should be in line with the design manual and should display in sentence case. 

c5c20af0-b9ea-11e5-9e87-7e0c4ef80d9b

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
The new FCM categories now align with what's in the design manual.
Fixes #1337 

## Testing
Create a sublanding page and check out the new categories in the Featured Content Module

## Preview
image

image

## Review
- @kave 
- @kurtw 
- @richaagarwal

### CODE REVIEW

File: cfgov/v1/models/ref.py [hunk 1]
```diff
@@ -121,6 +130,9 @@ def category_label(category):
             if category == slug:
                 return name
 
+def fcm_label(category):
+    for slug, name in fcm_types:
+            return name
```
Comment (kurtw) [inline cfgov/v1/models/ref.py]: This always returns the first `name` it sets! I think you're looking to return the name _if_ the `slug` and the given `category` are equal.

**Comment by kurtw** [comment]
This all looks great! Just address my comment before I can give a thumbs up.

File: cfgov/jinja2/v1/_includes/macros/category-icon.html [hunk 2]
```diff
@@ -27,7 +27,13 @@
...
         'blog': 'cf-icon-speech-bubble',
-        'newsroom': 'cf-icon-newspaper'
+        'newsroom': 'cf-icon-newspaper',
+        'featured event': 'cf-icon-date',
+        'featured blog': 'cf-icon-speech-bubble',
+        'featured video': 'cf-icon-play-round',
+        'featured tool': 'cf-icon-settings',
+        'featured news': 'cf-icon-newspaper',
+        'featured': 'cf-icon-favorite'
       }
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/macros/category-icon.html]: Can you add a todo to the top of this logic to eventually move it to the backend? The slug really should be passed in with the value instead.

File: cfgov/v1/models/ref.py (hunk 1 above)
Comment (kave) [inline cfgov/v1/models/ref.py]: so like
```
if slug == category:
   return name
```

File: cfgov/jinja2/v1/_includes/macros/category-icon.html (hunk 2 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/macros/category-icon.html]: There's an issue for it already. :)

File: cfgov/v1/models/ref.py (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/v1/models/ref.py]: Thanks for the catch!

**Comment by jimmynotjim** [comment]
:+1:

File: cfgov/jinja2/v1/_includes/macros/category-icon.html (hunk 2 above)
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/macros/category-icon.html]: Whoops, I keep forgetting

File: cfgov/v1/models/ref.py (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/v1/models/ref.py]: fixed in  a937940

**Comment by kurtw** [comment]
:+1:

**Pull request merged**